src/Tizen.Uix.Tts/Tizen.Uix.Tts/TtsClient.cs

```diff
@@ -848,6 +959,34 @@ public bool IsScreenReaderOn
             }
         }
 
+        /// <summary>
+        /// Sets the current playing mode.
+        /// </summary>
+        /// <since_tizen> 11 </since_tizen>
+        /// <value>
+        /// The current playing mode.
+        /// </value>
+        /// <feature>
+        /// http://tizen.org/feature/speech.synthesis
+        /// </feature>
+        /// <exception cref="InvalidOperationException">This exception can be due to an invalid state.</exception>
+        /// <exception cref="NotSupportedException">This exception can be due to TTS not supported.</exception>
+        /// <pre>
+        /// The Client must be in the <see cref="State.Created"/> state.
+        /// </pre>
+        public PlayingMode PlayingMode
```

Comment:
Dear, @wiertel .
I have a question about this property.
There is only a setter function for playing mode in native API, so I implemented only a setter for this property.
Is it okay to remain it or better to implement a getter?

Reply:
Having a write-only property is possible, but unusual. More fitting to cases when value is a secret, so should not be accessible. On one hand we prefer limit implementing logic in C# layer, but on the other hand it should not be problematic to implement getter for that property. If the default playing mode is defined in C API we could initialize the property to that value, if there is no default defined we would need to choose one. It's even more complicated if C API requires that the playing mode is set before other operations or there are some additional limitations/requirements.
My suggestion is to implement the getter if it is not complicated.

Reply:
Thank you for sharing your oppinion.
The default playing mode is defined as PlayingMode.ByServer in C API, so I think it is not hard to make getter for this property.
I will request review again after adding getter.